support/hololens/ServoApp/ServoControl/Servo.cpp

```diff
@@ -119,7 +119,7 @@ Servo::Servo(hstring url, hstring args, GLsizei width, GLsizei height,
 
   // Example Call when *pfilters[] is used:
   // o.vslogger_mod_list = pfilters; // servo log modules
-  // o.vslogger_mod_size = sizeof(pfilters) / sizeof(pfilters[0]) -1; //
+  // o.vslogger_mod_size = sizeof(pfilters) / sizeof(pfilters[0]); //
```

Comment:
Not sure why this has changed?

Reply:
I realized that it was wrong. The last filter was never used.